.travis.yml

```diff
@@ -10,7 +10,10 @@ addons:
   postgresql: "9.3"
 before_script:
   - gem install pg
-  - psql -c 'create database travis_ci_test;' -U postgres
+  - psql -c "CREATE USER ox_exchange WITH CREATEDB PASSWORD 'ox_exchange'" -U postgres
+  - psql -c 'CREATE DATABASE ox_exchange_test' -U postgres
+  - psql -c 'GRANT ALL PRIVILEGES ON DATABASE ox_exchange_test TO ox_exchange' -U postgres
+  - RAILS_ENV=test bundle exec rake db:migrate
 script:
   - RAILS_ENV=test bundle exec rake --trace
 branches:
```

Comment:
While this works, you could also export the travis defaults as environmental variables so Rails will use them when it reads database.yml

That way you can use standard rake tasks to create the db and everything

```yaml
before_script:
  - export OXE_DB_USER = postgres
  - export OXE_DB_PASS =
  - export OXE_TEST_DB =travis_ci_test
  - RAILS_ENV=test bundle exec rake db:create db:migrate db:seed
```

That's all untested of course, so take with a huge grain of salt and don't be afraid to tell me I'm crazy